inference-engine/src/mkldnn_plugin/nodes/mkldnn_conv_node.cpp

```diff
@@ -406,8 +407,9 @@ void MKLDNNConvolutionNode::initSupportedPrimitiveDescriptors() {
                 auto weightsPrc = MKLDNNExtensionUtils::IEPrecisionToDataType(dw_conv_in_dt == mkldnn_u8 ? Precision::I8 : Precision::FP32);
                 auto biasPrc = memory::data_type::f32;
 
-                MKLDNNDims dwWeightsDims({dw_conv_oc, (ptrdiff_t)1, (ptrdiff_t)1, dw_conv_kernel[Y_AXIS], dw_conv_kernel[X_AXIS]});
-                MKLDNNDims dwBiasesDims({dw_conv_oc});
+                std::vector<size_t> dwWeightsDims({static_cast<size_t>(dw_conv_oc), 1, 1, static_cast<size_t>(dw_conv_kernel[Y_AXIS]),
+                                                   static_cast<size_t>(dw_conv_kernel[X_AXIS])});
+                std::vector<size_t> dwBiasesDims({static_cast<size_t>(dw_conv_oc)});
```

Comment:
M.b. it is better to change the type of `dw_conv_oc` member than statically cast it everywhere?